aggregate_root/spec/aggregate_root_spec.rb

```diff
@@ -69,15 +69,15 @@
     order.apply(order_created)
     order.store(stream, event_store: event_store)
 
-    expect(event_store.read_stream_events_forward(stream)).to eq [order_created]
+    expect(event_store.read.stream(stream).each.to_a).to eq [order_created]
 
     restored_order = Order.new.load(stream, event_store: event_store)
     expect(restored_order.status).to eq :created
     order_expired = Orders::Events::OrderExpired.new
     restored_order.apply(order_expired)
     restored_order.store(stream, event_store: event_store)
 
-    expect(event_store.read_stream_events_forward(stream)).to eq [order_created, order_expired]
+    expect(event_store.read.stream(stream).each.to_a).to eq [order_created, order_expired]
```

Comment:
`each.to_a` look strange for me, full enumerable implementation maybe? (separate PR of course)